pkgs/development/tools/rust/rust-analyzer/generic.nix

```diff
@@ -1,11 +1,12 @@
-{ lib, stdenv, fetchFromGitHub, rustPlatform, darwin
-, useJemalloc ? false
-, doCheck ? true
-
 # Version specific args
-, rev, version, sha256, cargoSha256 }:
+{ rev, version, sha256, cargoSha256, ... }@args:
```

Comment:
Is it necessary to split the arguments? Looking at the output of `rg -B3 '@args'`, it doesn't look like a common pattern in nixpkgs.